src/commcare_cloud/ansible/deploy_keepalived.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: deploy keepalived
+  hosts: couchdb2_proxy
+  become: true
+  vars:
+     monit_process: "haproxy"
+  roles:
+    - role: keepalived
```

Comment:
this should have a `when` clause on it that check if there are 2 hosts in the `couchdb2_proxy` group